glue_solar/instruments/iris/stack_spectrograms.py

```diff
@@ -59,12 +59,20 @@ def stack_spectrogram_sequence(cube_sequence, memmap=True, reproject=False):
         if not reproject:
             memmap[i+1] = cube_sequence[i+1].data
         else:
-            reproject_interp((cube.data, cube.wcs),
-                             target_wcs, shape_out=target_shape,
-                             independent_celestial_slices=False,
-                             order=0,
-                             return_footprint=False,
-                             output_array=memmap[i+1])
+            try:
+                reproject_interp((cube.data, cube.wcs),
+                                 target_wcs, shape_out=target_shape,
+                                 hdu_in=0,
+                                 order=0,
+                                 return_footprint=False,
+                                 output_array=memmap[i+1])
+            except Exception:
```

Comment:
What was the Exception?

Reply:
Oh, it is for the reproject deprecation of `independent_celestial_slices`...

Reply:
Could I get a comment saying this please? Does it still error with the latest reproject?

Reply:
Sure, and should I restrict the reproject version to `> 0.6`?

Reply:
No, the error goes away with the latest version of `reproject`

Reply:
Yeah pin it to the version that works.

Reply:
Cool, pin `reproject` to version `>= 0.7` and remove redundant snippet of try-and-catch code.